### PR TITLE
Fix broken preset links; presets without versions

### PR DESCRIPTION
**Pull request by mattdangerw**

### CODE REVIEW

File: keras_hub/src/models/clip/clip_presets.py [hunk 1]
```diff
@@ -79,15 +79,4 @@
         },
         "kaggle_handle": "kaggle://keras/clip/keras/clip_vit_g_14_laion2b_s12b_b42k/1",
     },
-    "clip_vit_bigg_14_laion2b_39b_b160k": {
```
Comment (SamanehSaadat) [inline keras_hub/src/models/clip/clip_presets.py:82]: This model still exists on Kaggle but we want to remove it here? https://www.kaggle.com/models/kerashub/clip/keras/clip_vit_bigg_14_laion2b_39b_b160k

**Review by SamanehSaadat — APPROVED** [review]
Thanks, Matt!

File: keras_hub/src/models/clip/clip_presets.py (hunk 1 above)
Comment (mattdangerw) [reply keras_hub/src/models/clip/clip_presets.py:82]: Thanks! I think this was failing to download but maybe it was transient. Let me take a look here.

Comment (mattdangerw) [reply keras_hub/src/models/clip/clip_presets.py:82]: I see, this is on the staging org we were using https://www.kaggle.com/organizations/kerashub but not https://www.kaggle.com/organizations/keras. I'll move it over.

Comment (mattdangerw) [reply keras_hub/src/models/clip/clip_presets.py]: done!

**Pull request merged**